### ISSUE TITLE
OpenID IDP wizard not populating urls

### ISSUE THREAD

**Issue by konsri** [p2-inc/idp-wizard#178]
Steps to reproduce
1. Login to https://phasetwo.io/dashboard/deployments
2. Click on Open Console
3. Click on Organizations
4. Create organization
5. Click on organization link
6. Copy Organization Portal Link from Action button
7. Open this link in new window
8. Click on OpenID identity provider.
9. Provide details in steps
10. Finish the wizard
11. Goback to your admin UI.
12. Click on identity providers
13. Open the newly created identity provider
Expected
	Authorization/Token/Logout/User Info urls are populated
Issue
	All these urls are empty

**Comment by xgp** [comment]
@pnzrr need to check that the generic OIDC wizard is building the IdP config properly.

**Comment by pnzrr** [comment]
@konsri for step 9 what step are you using to provide details? There is a Configuration URL, File or Details. Are you using the manual one? 

I was able to reproduce your error with the URL method. I have updated the API call to include data returned from the URL validation call. 


---
@xgp For manual, it does look like all values are being passed correctly to the `import-config` endpoint but getting a 400 back from the server when trying to validate manually

<img width="1280" alt="image" src="https://github.com/p2-inc/idp-wizard/assets/93841792/f13b9467-2eb1-4183-9ea8-d077001230a9">
<img width="549" alt="image" src="https://github.com/p2-inc/idp-wizard/assets/93841792/bc24fd4d-1afd-494d-8a4c-5efdc54c5bd3">

This may be a separate issue we can file.
